src/output-json-ssh.c

```diff
@@ -75,25 +75,26 @@ static int JsonSshLogger(ThreadVars *tv, void *thread_data, const Packet *p,
         return 0;
     }
 
-    json_t *js = CreateJSONHeader(p, LOG_DIR_FLOW, "ssh", NULL);
+    JsonBuilder *js = CreateEveHeaderWithTxId(p, LOG_DIR_FLOW, "ssh", NULL, tx_id);
```

Comment:
we're creating a JsonBuilder here, so we should use only this one. The rust code should not create its own, but instead take this one as an argument and continue to add to it.

Reply:
ok